### PR TITLE
Math.test.ts

### PR DESCRIPTION
**Pull request by gcrev93**
Unit test for the math helper function.

Testing the randomElement function using SlotId Type.

### CODE REVIEW

**Comment by kevinleung23** [comment]
@gcrev93 Changing base from `master` -> `development`

**Comment by kevinleung23** [comment]
Wondering how we can test and actually prove the function is random? This is not used throughout the actual Wolf stages so this is not as critical.. but currently it's not proved it is indeed random. Is there a way we can provide a seed?

File: src/tests/unit/helpers/math.test.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import * as mathHelpers from '../../../../src/helpers/math'
+import { SlotId, AllSyncStorageLayer } from '../../../types';
+
+interface MockUserState {
```
Comment (howlowck) [inline src/tests/unit/helpers/math.test.ts]: Remove `MockUserState`

**Review by howlowck — CHANGES_REQUESTED** [review]

File: src/tests/unit/helpers/math.test.ts (hunk 1 above)
Comment (gcrev93) [reply src/tests/unit/helpers/math.test.ts]: removed

**Comment by gcrev93** [comment]
Just to close this PR, I removed MockState. Can we add the additional random testing as another task before closing this out? @wolf-packs/wolf-developers

**Review by dnastrain — APPROVED** [review]
:shipit:

**Review by howlowck — APPROVED** [review]

**Pull request merged**